Meta/Lagom/CMakeLists.txt

```diff
@@ -290,6 +302,11 @@ lagom_lib(TimeZone timezone
 )
 target_compile_definitions(LibTimeZone PRIVATE ENABLE_TIME_ZONE_DATA=$<BOOL:${ENABLE_TIME_ZONE_DATABASE_DOWNLOAD}>)
 
+# Disable previously enabled gcc extensions
+if (CYGWIN)
+    set(CMAKE_CXX_EXTENSIONS OFF)
+endif()
```

Comment:
I'd rather do this selective enabling of extensions with target properties on those specific libs, but we can refactor that late